apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationCommentList.jsx

```diff
@@ -1,18 +1,69 @@
-import React from 'react'
+import React, { Component } from 'react'
 import django from 'django'
 
 import ModerationComment from './ModerationComment'
 
-export default function (props) {
-  const byText = django.gettext('By ')
+export default class ModerationCommentList extends Component {
+  constructor (props) {
+    super(props)
 
-  return (
-    <div className="row mb-2">
-      <div className="col-12">
-        <h1 className="m-0">project title</h1>
-        <span className="text-muted">{byText}organisation</span>
-        <ModerationComment />
+    this.state = {
+      aiClassifications: [],
+      userClassifications: [],
+      isLoaded: false
+    }
+  }
+
+  componentDidMount () {
+    this.loadData()
+    this.timer = setInterval(() => this.loadData(), 3000)
+  }
+
+  async loadData () {
+    fetch(this.props.aiclassificationApiUrl)
+      .then(res => res.json())
+      .then(json => {
+        this.setState({
+          aiClassifications: json
+        })
+      }).catch((err) => {
+        console.log(err)
+      })
+    fetch(this.props.userclassificationApiUrl)
+      .then(res => res.json())
+      .then(json => {
+        this.setState({
+          userClassifications: json,
+          isLoaded: true
+        })
+      }).catch((err) => {
+        console.log(err)
+      })
+  }
+
+  componentWillUnmount () {
+    clearInterval(this.timer)
+    this.timer = null
+  }
+
+  render () {
+    const userClassifications = this.state.userClassifications
+    const byText = django.gettext('By ')
+
+    return (
+      <div className="row mb-2">
+        <div className="col-12">
+          <h1 className="m-0">project title</h1>
+          <span className="text-muted">{byText}organisation</span>
+          {userClassifications.map((item, i) => (
+            <li key={i}>
```

Comment:
@Rineee the <li> element is what is causing the bullet point for each comment as it assumes it's a list element so add default styling, we probably should do some juggling between this and the comment component maybe just make comment a div and rm <ul>and<li> and wrap the <li> elements in this component in a <ul>